### ISSUE TITLE
Border line join round does not work with BlAnnulusSector

### ISSUE THREAD

**Issue by Nyan11** [pharo-graphics/Bloc#515]
Hello,

Both shape in the image has joinRound in border.
We can see the round shape on the Square (Rectangle geometry) but not on the annulus.

![image](https://github.com/pharo-graphics/Bloc/assets/34318678/55d0be8f-6e8e-4044-b43e-a96223ee3f91)

**Comment by tinchodias** [comment]
Hello @Nyan11 and @labordep, it the objective (or scenario of use) the same as #516 ? I mean, if we fix that issue without these rounded corners, is this issue also fixed?

Becuase I plan to fix that issue with a new geometry that allows drawing an arc. And when you configure the border of that arc to have "rounded cap" then yopu have the desired visual effect.

**Comment by Nyan11** [comment]
Yes, it was the same usecase as in https://github.com/pharo-graphics/Bloc/issues/516 .
